src/store/actions.ts

```diff
@@ -897,42 +897,50 @@ export const parseSynonymNames = (json: { names: [string], exactNames: [{ name:
 }
 
 export const getQuickSearch = async ({ commit, getters }, cleanedName: {exactMatch: string, synonymMatch: string}) => {
-  try {
-    commit('mutateDisplayedComponent', 'QuickSearchPending')
-    let encodedAuth = btoa(`${window['quickSearchPublicId']}:${window['quickSearchPublicSecret']}`)
+  const quickSearchPublicId = window['quickSearchPublicId']
+  const quickSearchPublicSecret = window['quickSearchPublicSecret']
 
-    const tokenResp = await axios.post(window['authTokenUrl'], 'grant_type=client_credentials', {
-      headers: { Authorization: `Basic ${encodedAuth}`, 'content-type': 'application/x-www-form-urlencoded' }
-    })
+  // only do quick search if we have id and secret
+  if (quickSearchPublicId && quickSearchPublicSecret) {
```

Comment:
This is a fix to prevent an error when the quick search keys aren't populated (ie, when running locally). So it just skips this code.

I made this same fix to the old code in New Request Module.

Reply:
(The diff here is mostly whitespace due to indentation.)

Reply:
ah was wondering about this one and why it didn't make the request locally. 👍